### PR TITLE
Change goldbar text for updating keyboard bindings

### PR DESCRIPTION
**Pull request by chborl**

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs [hunk 1]
```diff
@@ -191,15 +193,18 @@ private void ShowGoldBar()
...
             infoBarService.ShowInfoBarInGlobalView(
                 string.Format(message, ReSharperExtensionName),
-                new InfoBarUI(title: ServicesVSResources.Restore_Visual_Studio_keybindings,
+                new InfoBarUI(title: ServicesVSResources.Reset_Visual_Studio_default_keymapping,
                               kind: InfoBarUI.UIKind.Button,
                               action: RestoreVsKeybindings),
-                new InfoBarUI(title: ServicesVSResources.Use_Keybindings_for_extensions,
+                new InfoBarUI(title: string.Format(ServicesVSResources.Apply_0_keymapping_scheme, ReSharperKeyboardMappingName),
+                              kind: InfoBarUI.UIKind.Button,
+                              action: OpenExtensionsHyperlink),
```
Comment (JieCarolHu) [inline src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs]: same action for this apply ReSharperKeyboardMapping and VSCodeKeyboardMapping？

Comment (chborl) [reply src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs]: Yes, for now they will both go to the same FWLink and each button might later be updated to make the specific change for the user.

**Comment by 333fred** [comment]
@chborl can we get a screenshot of what this change looks like? I'm concerned that the length of the strings will look bad on standard resolution monitors.

**Comment by chborl** [comment]
@333fred , here's a screenshot of the change.  image

**Comment by chborl** [comment]
@333fred , I changed my settings to 1080p and the screenshot looks like this
image

**Comment by chborl** [comment]
@333fred  and here's a narrow screen
image

**Comment by heejaechang** [comment]
@kuhlenh can you review whether wording looks good to you?

**Comment by chborl** [comment]
@heejaechang , the wording for the goldbar came directly from Kasey in email so it should be good to go.

**Comment by jinujoseph** [comment]
Approved to merge for 15.8.Preview5

**Pull request merged**